src/server/game/Handlers/MovementHandler.cpp

```diff
@@ -319,22 +319,25 @@ void WorldSession::HandleMovementOpcodes(WorldPacket & recvData)
 
     recvData.readPackGUID(guid);
 
-    MovementInfo movementInfo;
-    movementInfo.guid = guid;
-    ReadMovementInfo(recvData, &movementInfo);
-
-    recvData.rfinish();                         // prevent warnings spam
-
     // pussywizard: typical check for incomming movement packets
     if (!mover || !mover->IsInWorld() || mover->IsDuringRemoveFromWorld() || guid != mover->GetGUID())
+    {
+        recvData.rfinish();
         return;
+    }
+
+    MovementInfo movementInfo;
+    movementInfo.guid = guid;
+    ReadMovementInfo(recvData, &movementInfo);
 
     if (!movementInfo.pos.IsPositionValid())
     {
         recvData.rfinish();                     // prevent warnings spam
         return;
     }
 
+    recvData.rfinish();                         // prevent warnings spam
```

Comment:
this is true ?

recvData It should finish when all bets have also been reviewed.
this line must move to between lines 504-505

Reply:
@LichKing255 please open a PR